.vscode/settings.json

```diff
@@ -1,8 +1,6 @@
 // Place your settings in this file to overwrite default and user settings.
 {
-  "editor.renderWhitespace": "boundary",
```

Comment:
Once you update VS Code to the latest version, there will be vertical lines marking white space by default. That seems fine to me. And editorconfig mostly solves the tabs/spaces issue.